packages/react-reconciler/src/ReactFiberHooks.js

```diff
@@ -2259,6 +2259,9 @@ function rerenderFormState<S, P>(
     );
   }
 
+  // For mount, pending is always false.
+  const [isPending] = rerenderState(false);
```

Comment:
Since we don't actually need the result, all we need to do is advance the hook list.

```suggestion
  updateWorkInProgressHook();
```

Reply:
[`e32f38a` (#28557)](https://github.com/facebook/react/pull/28557/commits/e32f38a05b1861af1be67ce223dafbf5f83e4799)

Reply:
Feels a bit brittle but you know this part better than I do.

Reply:
I consider this pretty low level code so I think it's fine. I care more about not observing the value of `isPending` since it causes a bunch of redundant read operations.